content/en/tracing/faq/profiling_migration.md

```diff
@@ -0,0 +1,183 @@
+| `DD_API_KEY`                                     | String        | Deprecated in version 0.39. The [Datadog API key][1] to use when uploading profiles. See above for how to configure dd-trace-java.jar to upload profiles via the Datadog Agent instead. Supported in version 0.37.                                        |
+| `DD_PROFILING_API_KEY`                           | String        | Deprecated in version 0.39. The [Datadog API key][1] to use when uploading profiles. |
+| `DD_SITE`                                        | String        | Deprecated in version 0.39. If your organization is on Datadog EU site, set this to `datadoghq.eu`. See above for how to configure dd-trace-java.jar to upload profiles via the Datadog Agent instead.                          |
+| `DD_PROFILING_TAGS`                              | String        | Deprecated in 0.38 in favor of `DD_TAGS`. Tags to apply to an uploaded profile. Must be a list a `key:value` comma separated list like: `<KEY1>:<VALUE1>,<KEY2>:<VALUE2>`. |
+| DD_PROFILING_PROXY_HOST                          | String        | Deprecated in version 0.39. Host for your proxy (`my-proxy.example.com`).    |
+| DD_PROFILING_PROXY_PORT                          | String        | Deprecated in version 0.39. Port used by your proxy. Default port is `8080`. |
+| DD_PROFILING_PROXY_USERNAME                      | String        | Deprecated in version 0.39. Username used by your proxy.                     |
+| DD_PROFILING_PROXY_PASSWORD                      | String        | Deprecated in version 0.39. Password used by your proxy.                     |
+
+[1]: https://app.datadoghq.com/account/settings#agent/overview
+[2]: https://app.datadoghq.com/apm/install
+{{% /tab %}}
+
+{{% tab "Go" %}}
+
+Perform following steps to migrate your service to send profiles directly through the agent:
+
+1. Upgrade your agent to version [7.20.2][1]+ or [6.20.2][1]+
+
+3. Clear `DD_API_KEY` or `DD_PROFILING_API_KEY` environment variables as they were deprecated in version 0.55:
+
+    ```shell
+    unset DD_PROFILING_API_KEY
+
+    # OR
+
+    unset DD_API_KEY
+    ```
+
+2. Upgrade the tracing library to [version 1.25.0][2]+:
+
+    ```shell
+    go get gopkg.in/DataDog/dd-trace-go.v1
+    ```
+
+4. Ensure that you have imported the [profiler][1] at the start of your application:
+
+    ```Go
+    import "gopkg.in/DataDog/dd-trace-go.v1/profiler"
+    ```
+
+5. To profile your code, add a [Datadog API key][2], set your environment, service, and version, then start the profiler:
+
+    ```Go
+    err := profiler.Start()
+    if err != nil {
+        log.Fatal(err)
+    }
+    defer profiler.Stop()
+    ```
+
+    **NOTE** Profilers must now be configured through environment variables instead of code. See below for deprecated methods in the tracer version 1.25.0.
```

Comment:
Is this actually true? Why the deprecation?

Reply:
It is not, ive reverted this on Go actually :)